test/abi/macOS/arm64/synchronization.swift

```diff
@@ -643,3 +619,28 @@ Added: _$ss9UnmanagedVyxG15Synchronization27AtomicOptionalRepresentableADWP
 
 // protocol conformance descriptor for <A where A: Synchronization.AtomicOptionalRepresentable> A? : Synchronization.AtomicRepresentable in Synchronization
 Added: _$sxSg15Synchronization19AtomicRepresentableA2B0b8OptionalC0RzlMc
+
+// protocol conformance descriptor for < where A: ~Swift.Copyable> Swift.UnsafePointer<A> : Synchronization.AtomicRepresentable in Synchronization
+Added: _$sSPyxG15Synchronization19AtomicRepresentableABRiczrlMc
+
+// protocol witness table for < where A: ~Swift.Copyable> Swift.UnsafePointer<A> : Synchronization.AtomicRepresentable in Synchronization
+Added: _$sSPyxG15Synchronization19AtomicRepresentableABRiczrlWP
+
+// protocol conformance descriptor for < where A: ~Swift.Copyable> Swift.UnsafePointer<A> : Synchronization.AtomicOptionalRepresentable in Synchronization
+Added: _$sSPyxG15Synchronization27AtomicOptionalRepresentableABRiczrlMc
+
+// protocol witness table for < where A: ~Swift.Copyable> Swift.UnsafePointer<A> : Synchronization.AtomicOptionalRepresentable in Synchronization
+Added: _$sSPyxG15Synchronization27AtomicOptionalRepresentableABRiczrlWP
+
+// protocol conformance descriptor for < where A: ~Swift.Copyable> Swift.UnsafeMutablePointer<A> : Synchronization.AtomicRepresentable in Synchronization
+Added: _$sSpyxG15Synchronization19AtomicRepresentableABRiczrlMc
+
+// protocol witness table for < where A: ~Swift.Copyable> Swift.UnsafeMutablePointer<A> : Synchronization.AtomicRepresentable in Synchronization
+Added: _$sSpyxG15Synchronization19AtomicRepresentableABRiczrlWP
+
+// protocol conformance descriptor for < where A: ~Swift.Copyable> Swift.UnsafeMutablePointer<A> : Synchronization.AtomicOptionalRepresentable in Synchronization
+Added: _$sSpyxG15Synchronization27AtomicOptionalRepresentableABRiczrlMc
+
+// protocol witness table for < where A: ~Swift.Copyable> Swift.UnsafeMutablePointer<A> : Synchronization.AtomicOptionalRepresentable in Synchronization
+Added: _$sSpyxG15Synchronization27AtomicOptionalRepresentableABRiczrlWP
+
```

Comment:
I suspect you'll need to edit the exception list for synchronization on x86_64 as well.

Reply:
Huh, why don't have to do this for `UnsafeBufferPointer`? 🤔

Reply:
oh you're right we do need to edit the buffer pointer definitions in synchronization too

Reply:
I did not notice any test failures for those! Something might be amiss

Reply:
Yeah, I meant the actual source for the conformances need to account for suppressing pointee's copyability for the buffer pointers now.

Reply:
Duh! 🤦‍♂️